atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -30,13 +28,4 @@
 
     @Override
     boolean isInitialized();
-    /**
-     * Returns a sorted list of row keys in the specified range.
-     *
-     * @param tableRef table for which the request is made.
-     * @param startRow inclusive start of the row key range. Use empty byte array for unbounded.
-     * @param endRow inclusive end of the row key range. Use empty byte array for unbounded.
-     * @param maxResults the request only returns the first maxResults rows in range.
-     */
-    List<byte[]> getRowKeysInRange(TableReference tableRef, byte[] startRow, byte[] endRow, int maxResults);
```

Comment:
This might break users of CassandraKVS that access this method through certain kinds of reflection 😅